tokens/o2.json

```diff
@@ -552,25 +544,45 @@
       "type": "color",
       "description": "darkModeO2BluePrimaryDark"
     },
```

Comment:
missing tones of constant here!
![image](https://user-images.githubusercontent.com/6722153/201371119-42ddde8d-4275-4aeb-a9d4-64aca73886d0.png)